### DISCUSSION TITLE
Blitzwolf SHP6 Pro new design

### DISCUSSION THREAD

**Discussion by michaelpiron** [arendst/Tasmota#15698] [Q&A] [answered]
Hi all,

recently bought some extra Blitzwolf SHP6 Pro plugs, but it seems the design of the plug has changed. Anyone has experience in how to open the plug to flash new firmware? I hope the plug is still powered by EsP8266.

there is no longer a bolt in the earthing hole of the plug

![B4C56714-6CCA-4F7B-93D4-9600594E8E88](https://user-images.githubusercontent.com/4469877/170465684-5c0007be-ae1c-4bbb-8177-f9e1e1e5c3d5.jpeg)
![88ABEC03-CCEA-41CC-BDED-52E6CDFE76B6](https://user-images.githubusercontent.com/4469877/170465717-3a8705ba-732d-4ae5-970f-d9cab7afcd53.jpeg)
![DFABE1AB-AD2E-4727-B4BA-D3AA1FA1BE7F](https://user-images.githubusercontent.com/4469877/170465722-5eed6054-1ffc-4aa3-b12b-2dab0c5b94dd.jpeg)

![image](https://user-images.githubusercontent.com/4469877/170466009-572b3749-bac6-4466-919b-18f1ce47c228.jpg)

**Comment by barbudor** [comment]
Most probably the inner transparent part is glued and you may be able to take the glue apart by pressing all around the plug in a wise

Alternatively, if you still have an opportunity to cancel the sale and get Nous A1 instead, AFAIK these one still have the screen in the earth hole and running ESP8266
By 4 : https://www.amazon.fr/dp/B0054PSES6
By 2 : https://www.amazon.fr//dp/B07WPWPVLZ

**Reply by Nous-smart-home** [reply]
> Most probably the inner transparent part is glued and you may be able to take the glue apart by pressing all around the plug in a wise
> 
> Alternatively, if you still have an opportunity to cancel the sale and get Nous A1 instead, AFAIK these one still have the screen in the earth hole and running ESP8266 By 4 : https://www.amazon.fr/dp/B0054PSES6 By 2 : https://www.amazon.fr//dp/B07WPWPVLZ

Hello, unfortunately regular Nous A1 no longer supports Tasmota, its now has more advanced chip inside and its positioned more for the casual Tuya/Nous app users, but we launched Nous A1T smart plug with Tasmota already preflashed(they sold out for now, but it will be back in July)
And you can find it here:
https://www.amazon.de/dp/B0054PSI46

**Reply by barbudor** [reply]
Thanks @Nous-smart-home for letting us know
I hope you will get them back in stock soon and keep them in the catalog for some time.

**Comment by michaelpiron** [comment, marked as answer]
The verdict: the new Blitzwolf SHP6 Pro no longer has an ESP chip inside…

**Reply by barbudor** [reply]
Condoléances

**Reply by marekmarecki** [reply]
Thanks. I almost bought it

**Comment by michaelpiron** [comment]
It would make sense to warn people that the new SHP6 Pro no longer has ESP, and hence no longer is compatible with Tasmota.
Could someone mention this on the templates page? -> https://templates.blakadder.com/blitzwolf_SHP6-15A.html
@blakadder

**Reply by mcanteri** [reply]
Just bought 2 of them and they are useless ...

**Reply by barbudor** [reply]
The doc is open source and can be edited by anyone
Don't wait expecting someone does that for you

**Comment by tjheikki** [comment]
Also got four of these useless things now, looking for ways to get them in use.
Rather tight but just might be possible to replace the "W701" board by an ESP one, like here: https://github.com/c-klinger/Gosund-ESPMod
Or, install another open firmware (not Tasmota but got similar features) like this: https://github.com/openshwprojects/OpenBK7231T_App
This "new SHP6" BTW isn't that difficult to open:
- Pry out the transparent part with a knife or flat steel opening tool
- Unscrew the 3 screws
- Unsolder the ground pin (you'll see when there)
- Unscrew 2 more screws
- Push out the innards by the AC pins
- Unclip the white plastic fixture, detach the antenna connector
![image](https://github.com/arendst/Tasmota/assets/16526467/5e027419-b27e-4fcf-8d98-3c904b014eaa)

**Reply by Gohico** [reply]
LOL, I got a package with 4 SHP6-Pro today as well - with the false assumption they still have ESPs.
I'll most likely don't have time in the next weeks but if you happen to find a way please share it. 
In the mean time I'm trying to get one of mine open (I'd say the transparent plastic is glued in - at least in my case - you can see the glue on the sides). 
Argh...

**Reply by Jason2866** [reply]
Looks like a MCU daugther board design where an ESP8266 compatible one exists

**Reply by devsnd** [reply]
I also accidentally bought 4 of the shp6 pro. 

Your disassembly instructions are very helpful, but optimistic and I had to put up quite a fight to open the thing.

![DSC_3824~2](https://github.com/arendst/Tasmota/assets/1980794/a7aae31b-da66-4f13-af77-6ecabaa19aec)

Here's what I found inside. Instead of w701 it says BL0937. Can't find anything about a chip with that name... 🤔

**Reply by Jason2866** [reply]
BL0937 is the powering monitoring chip. Seems the combine on the daughterboard more things. On the other side is probably the wifi chip. Makes a replacement nearly impossible.

**Reply by devsnd** [reply]
Whoops, I'm doofus.

It looks like the same once I am looking at the correct side of the PCB 🤦

I guess it says w701

![20230922_215408~2](https://github.com/arendst/Tasmota/assets/1980794/187fb7a5-439f-4036-834a-c635f2afd071)

**Reply by devsnd** [reply]
Oh, and for those that keep wondering what step 3 is about (unsolder ground pin)

This is what the two parts look like unsoldered. It's easy once you know...

![DSC_3831~2](https://github.com/arendst/Tasmota/assets/1980794/1686af19-4451-45ab-bdc3-dbde2b7df18f)

**Reply by sfromis** [reply]
Yeah, W701 means no Tasmota, and no easy module swap
